dev_requirements.txt

```diff
@@ -6,4 +6,5 @@ black;python_version >= '3.6'
 aioresponses>=0.6.2
 pytest-asyncio>=0.12.0
 azure-core>=1.11.0
-asgiref>=3.2.3
\ No newline at end of file
+asgiref>=3.2.3
+tenacity>=8.0.0
```

Comment:
You don't want to add this here, this is dev requirements - meaning dependencies you only need when working on the repo.
You want to put it in azure-kusto-ingest's setup.py

Reply:
ok fixed.